src/components/UpcomingEventCard.vue

```diff
@@ -224,24 +267,78 @@ export default {
 	.text {
 		font-size: 0.75rem;
 		font-weight: 500;
-		line-height: 0.87rem;
 	}
 
 	.just-sb {
 		justify-content: space-between;
 	}
 
 	.link {
-		text-decoration: underline;
+		padding-left: 1.25rem;
+	}
+
+	.talk-link {
+		padding-left: 1rem;
 	}
 
 	.description-text {
-		color: #595959;
+		color: #010564;
 		line-height: 1rem;
+		margin-bottom: 0.75rem;
 	}
 
 	.vue-avatar--wrapper span {
 		margin-left: 0;
 	}
+
+	h3 {
+		margin-top: 0;
+	}
+
+	h4 {
+		font-weight: 500;
+		color: #010564;
+		line-height: 2.5rem;
+	}
+
+	.participant {
+		padding-top: 0.25rem;
+	}
+
+	.email-icon::before {
```

Comment:
Instead of having a lot of repeated CSS between these icons, you should create a shared class, and then have the shared rules be under that class.

You can unify the padding by setting it to:
```
padding: 0.5rem 0.8rem;
```

And to have the beginning of each icon line up nicely you can change the background-position to
```
background-position: left;
```

Lastly it seems like `position: relative;` isn't needed.